meson.build

```
@@ -1,8 +1,9 @@
-project('gecko-camera', 'cpp',
-  version : '0.1',
-  default_options : [ 'warning_level=1',
-                      'buildtype=debugoptimized',
-                      'cpp_std=c++17'])
+project(
+  'gecko-camera',
+  'cpp',
+version : '0.1.0+fix.20210907141353.462b100-1',
+default_options : ['warning_level=1', 'buildtype=debugoptimized', 'cpp_std=c++17']
```

I think these just need some indentation.
```suggestion
  version : '0.1.0+fix.20210907141353.462b100-1',
  default_options : ['warning_level=1', 'buildtype=debugoptimized', 'cpp_std=c++17']
```

---

This also was committed by mistake. Fixed, thanks.

---

Thanks, the new version makes more sense, but the indentation still seems to be missing, if I'm not mistaken?

---

This is what `meson.build` looks like after it has been rewritten with the meson rewrite function, but I agree that it should be human readable.  Fixed properly now.

---

Oh, the machine-re-writing explains it, thanks.